features/networking/sctp.feature

```diff
@@ -18,6 +18,13 @@ Feature: SCTP related scenarios
       | ["spec"]["nodeName"]      | <%= cb.workers[0].name %> |
     Then the step should succeed
     And the pod named "sctpserver" becomes ready
+    And I wait up to 20 seconds for the steps to pass:
+    """
+    When I execute on the "sctpserver" pod:
+      | bash | -c | which nc |
```

Comment:
better to run `nc --version` or `nc --help` so we check if it actually works and cut-out the middle-man.

Reply:
@rbbratta good point, but if we are going to make a generic step https://github.com/openshift/verification-tests/pull/1663/files#r524897570 then I'm not sure this `--version` and `--help` option will apply to all commands.

Reply:
New commit remove above steps, no need to check which nc or nc version

Reply:
@pruan-rht Thanks for suggestion but yea we don't use such steps throughout the networking tests until these ones and it will be taken into consideration in case we tend to use such steps more later as that would make sense. For now I believe recent Weibin change should suffice the requirement.